monitoring/monitoring.go

```diff
@@ -235,16 +305,12 @@ func (s *Service) InstantQuery(query string) (string, error) {
 	}
 	defer errcapture.ExhaustClose(&err, res.Body, "metrics response")
 
-	body, err := ioutil.ReadAll(res.Body)
+	body, err := io.ReadAll(res.Body)
 	return string(body), err
 }
 
-func (s *Service) GetMonitoringRunnable() e2e.InstrumentedRunnable {
-	return s.p.InstrumentedRunnable
```

Comment:
Removing this breaks the meta monitoring E2E test so probably we need to add it back.